===================================================================
RCS file: /cvs/src/src/winsup/cygwin/include/paths.h,v
retrieving revision 1.6
diff -u -p -r1.6 paths.h
--- src/winsup/cygwin/include/paths.h	2 Nov 2010 17:38:36 -0000	1.6
+++ src/winsup/cygwin/include/paths.h	19 Aug 2011 20:08:11 -0000
@@ -17,10 +17,12 @@ details. */
 #define _PATH_DEV	"/dev/"
 #define _PATH_DEVNULL	"/dev/null"
 #define _PATH_LASTLOG	"/var/log/lastlog"
+#define _PATH_MAILDIR	"/var/spool/mail/"
 #define _PATH_MAN	"/usr/share/man"
 #define _PATH_MEM	"/dev/mem"
 #define _PATH_MNTTAB	"/etc/fstab"
 #define _PATH_MOUNTED	"/etc/mtab"
+#define _PATH_SHELLS	"/etc/shells"
 #define _PATH_STDPATH	"/bin:/usr/sbin:/sbin"
 #define _PATH_TMP	"/tmp/"
 #define _PATH_TTY	"/dev/tty"
